blog/index.html

```diff
@@ -134,85 +134,85 @@
 li#menu-item-9062 a:before {
     top: 0;
 }</style>
-					<style type="text/css">/* ***blog*** */
-
-pre code {
-        overflow: scroll !important;
-	white-space: pre !important;
-	font-size: 11px
-}
-
-header.side nav>ul>li {
-    padding: 15px !important;
-}
-
-.cnt {
-    margin-top: 50px !important;
-}
-
-.cnt a.customHref{
-    color: #f82 !important;
-}
-
-.bSe ul li:before {
-    content: '' !important;
-    position: absolute;
-    left: 0;
-    top: -2px;
-}
-
-.bSe ul {
-    list-style-type: disc !important;
-}
-
-.bSe table tbody tr{
-    border-top-width:0px !important;
-}
-
-code {
-    font-size: 0.9em !important;
-}
-
-pre{
-    font-family: Monaco, MonacoRegular, monospace !important;
-    font-size: 14px !important;
-    background-color: rgb(250,253,255) !important;
-    padding: 0px 3px 0px 3px !important;
-    white-space: pre !important;
-}
-
-.bSe h1, .bSe .entry-title {
-    line-height: 1.2;
-    font-size: 40px;
-}
-
-.bSe h2 {
-    font-weight: 300;
-}
-
-.bSe h2, .tve-woocommerce .bSe h2 {
-    font-family: "Roboto",sans-serif;
-}
-
-.cnt p, .tve-woocommerce .product p, .tve-woocommerce .products p {
-    font-family: Roboto,sans-serif;
-    font-weight: 300;
-}
-
-p {
-    margin-bottom: 15px;
-}
-
-
-header.fbh {
-    height: 49px;
-}
-header #logo>a>img {
-    max-width: 150px;
-}
-
-.gist .blob-code .blob-code-inner .js-file-line  {
-    white-space: pre !important;
+					<style type="text/css">/* ***blog*** */
```

Comment:
weird indentation here

Reply:
Fixed in https://github.com/StackStorm/stackstorm.com/pull/13/commits/703010b449637506fb390dd7613157ed8a573625